app/views/assessments/_results_panel.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class="result-summary">
+  <div class="result-pane result-fixed">
+    <div class="feedback-header">
+      <h3>Results</h3>
+    </div>
+    <div id="result-body">
+      <% if !@scoreHash.nil? %>
+        <h4>Autograder Scores</h4>
+        <table class="ui unstackable table">
+          <tbody>
+          <% @scoreHash.each do |key, value| %>
+            <% if key != '_total' %>
+              <tr>
+                <td><%= key %>:</td>
+                <td><%= value %></td>
```

Comment:
Nit: value should minimally be rounded to 1 d.p. to be consistent with the submission summary display (`value.to_f.round(1)`)

Same with the autograding total below